bindings/rust/wasmedge-sys/Cargo.toml

```diff
@@ -21,11 +21,14 @@ rand = "0.8.4"
 thiserror = "1.0.30"
 wasmedge-macro = {path = "../wasmedge-macro", version = "0.1"}
 wasmedge-types = {path = "../wasmedge-types", version = "0.3"}
-
+wasmtime-fiber = "1"
```

Comment:
Looks like we should have an empty line before `[build-dependencies]`.

Reply:
Sure